live-examples/css-examples/pseudo-class/autofill.css

```diff
@@ -3,10 +3,10 @@ label {
   margin-top: 1em;
 }
 
-input:autofill {
+input:-webkit-autofill {
   border: 3px solid darkorange;
 }
 
-input:-webkit-autofill {
+input:autofill {
```

Comment:
this might be a good time to declared these together with the webkit inside a forgiving selector list; using :is() or :where()

Reply:
Good call, so 

```css
input:is(:-webkit-autofill, :autofill) {
  border: 3px solid darkorange;
}
```

Reply:
done in https://github.com/mdn/interactive-examples/pull/2812/commits/7a90f9bc2eabc8c881bc0f1429ad04631d8e9417